pkgs/servers/calibre-web/default.nix

```diff
@@ -63,6 +63,8 @@ python.pkgs.buildPythonApplication rec {
     # and exit. This is gonna be used to configure calibre-web declaratively, as most of its configuration parameters
     # are stored in the DB.
     ./db-migrations.patch
+    # environ in tornado.wsgi.WSGIContainer no longer a static method from 6.3 version
+    ./static_environ.patch
```

Comment:
I think the patch is the right way, but since this is an application, how about pinning the tornado like sqlalchemy?
If upstream takes a long time to support tornado 6.3, it may be a pain to maintain the patches.

Reply:
I think we can merge this anyway,  since it's a broken package now.  And any better solution can be discussed in new PRs which trying to remove patches